tests/svrunit/suites/dev/6.6.6.0.xml

```diff
@@ -8,7 +8,7 @@
         <testsuite name="dockware/dev:6.6.6.0, Core Checks" group="core" dockerImage="dockware/dev:6.6.6.0">
             <directory>./../../tests/shared/base</directory>
             <directory>"./../../tests/packages/ubuntu/22"</directory>
-            <directory > ./../../tests/shared/base-6.5</directory> 
```

Comment:
hi there
not quite sure about this? did you manually change it or was is generated?

Reply:
This was generated via the command `make generate-tests` inside the `build` directory.

Reply:
I have adjusted these manually now

Reply:
ok perfect, ill check it out next time, seems weird

thanks again for this

Reply:
@boxblinkracer 
I have added a PR for that in https://github.com/dockware/dockware/pull/233
It was a check that checked for 6.5 in the string, which worked for 6.6.5.x but not for 6.6.6.1